config/vlan.py

```diff
@@ -50,6 +51,34 @@ def del_vlan(db, vid):
         db.cfgdb.set_entry('VLAN_MEMBER', k, None)
     db.cfgdb.set_entry('VLAN', 'Vlan{}'.format(vid), None)
 
+def restart_ndppd():
+    docker_exec_cmd = "docker exec -it swss {}"
+    ndppd_config_gen_cmd = "sonic-cfggen -d -t /usr/share/sonic/templates/ndppd.conf.j2,/etc/ndppd.conf"
+    ndppd_restart_cmd = "supervisorctl restart ndppd"
+
+    clicommon.run_command(docker_exec_cmd.format(ndppd_config_gen_cmd), display_cmd=True)
+    sleep(1)
+    clicommon.run_command(docker_exec_cmd.format(ndppd_restart_cmd), display_cmd=True)
+
+
+@vlan.command('proxy_arp')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.argument('mode', metavar='<mode>', required=True, type=click.Choice(["enabled", "disabled"]))
+@clicommon.pass_db
+def config_proxy_arp(db, vid, mode):
+    """Configure proxy ARP for a VLAN"""
+
+    log.log_info("'setting proxy ARP to {} for Vlan{}".format(mode, vid))
+
+    ctx = click.get_current_context()
+
+    vlan = 'Vlan{}'.format(vid)
+
+    if not clicommon.is_valid_vlan_interface(db.cfgdb, vlan):
+        ctx.fail("Interface {} does not exist".format(vlan))
+
+    db.cfgdb.set_entry('VLAN_INTERFACE', vlan, {"proxy_arp": mode})
```

Comment:
Please check if `mode == enabled/disabled`, if not print a message on the accepted values

Reply:
Click already checks if it is either enabled/disabled, see line 66